### DISCUSSION TITLE
Anyone thinking about DynamoDB, OpenSearch or AWS QLDB?

### DISCUSSION THREAD

**Discussion by liesislukas** [prisma/prisma#13758] [Q&A] [answered]
I wonder is there anyone thinking about DynamoDB, OpenSearch or AWS QLDB within prisma?

While SQL or mongo are great starting points for most of the apps, yet the best ones usually have some special features. Would like to see at least some progress to handle more specific use cases. E.g. if you want to have great search option in app, SQL/Mongo are very limited or experience performance bottlenecks relatively quickly, can't compete with OpenSearch. DynamoDB is amazing for high-speed low-size stuff, like e.g. analytics tracking, again SQL/mongo can do similar but it's just all different league of performance and manageability. QLDB make database immutable, so you can compile with higher standards that data is not modified in any way even with admin access, it's not even possible with sql or mongo.

https://aws.amazon.com/dynamodb
https://aws.amazon.com/qldb/
https://aws.amazon.com/opensearch-service/

**Comment by janpio** [comment, marked as answer]
For DynamoDB we have a feature request here: https://github.com/prisma/prisma/issues/1676
I am not aware of feature requests issue for the other ones - feel free to open them!

Our focus right now is not to build additional connectors to databases though, but build out the feature set of the current ones so people are really happy with what we offer there. When we have achieved that, we will tackle the most popular or useful requests additional connectors. (DynamoDB is on top of that list by popularity on GitHub issue upvotes)

**Reply by liesislukas** [reply]
Great, thank you for focusing on one thing at a time.

**Reply by liesislukas** [reply]
Created #13802 for OpenSearch
And #13803 for QLDB

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
